pages/shared/syncHelpers.js

```diff
@@ -43,6 +43,7 @@ module.exports.pullAndUpdate = function(locals, callback) {
         // update the content.
 
         const syncStage1A = function() {
+            const branch = locals.course.branch || config.cloneCourseDefaultBranch;
```

Comment:
In all of this code, we can assume that `locals.course.branch` is valid, so we can drop all the `||` logic.

Reply:
Done. Tested. No issues.